java/ql/src/experimental/Security/CWE/CWE-927/SensitiveBroadcast.ql

```diff
@@ -1,5 +1,5 @@
 /**
- * @name Use of Implicit Intent for Sensitive Communication
+ * @name Broadcasting sensitive data to all Android applicationss
```

Comment:
```suggestion
 * @name Broadcasting sensitive data to all Android applications
```

Reply:
Oops! Thanks for pointing out the typo. I've fixed it.